docs/content/dev/design_docs/api_auth.md

```diff
@@ -0,0 +1,54 @@
+# API Authentication
+
+## Public API
+All requests to the public API require authentication using a [bearer token](https://datatracker.ietf.org/doc/html/rfc6750).
+
+This token can be generated using the profile page in the frontend
+(which in turn uses the private API to generate the token).
+
+### Token Generation
+
+When a new token is requested via the private API, the backend generates a 64 bytes-long secret of
+cryptographically secure data and returns it as a base64url-encoded string, along with an identifier.
+That string can then be used by clients as a bearer token.
+
+A SHA-512 hash of the secret is stored in the database. To validate tokens, the backend computes the hash of the provided
+secret and checks it against the stored hash for the provided identifier.
+
+#### Choosing a hash function
+Unfortunately, there does not seem to be any explicit documentation about our exact use-case.
```

Comment:
```suggestion

Unfortunately, there does not seem to be any explicit documentation about our exact use-case.
```

Reply:
What should be changed here? You introduce a new blank line, resulting in two blank lines being below the heading...